erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -455,6 +460,42 @@ def validate_serial_no_based_delivery(self):
 				Item {0} is added with and without Ensure Delivery by \
 				Serial No.").format(item.item_code))
 
+def validate_inter_company_party(doctype, party, company, inter_company_order_reference):
```

Comment:
These functions can be commonified with invoice

Reply:
Fixed